mzLib/UsefulProteomicsDatabases/ProteinDbWriter.cs

```diff
@@ -7,6 +7,7 @@
 using System.Xml;
```

Comment:
should this be called "ProteinDbWriter"?

Reply:
It should probably be called BioPolymerDbWriter. Should this change be made in this PR?